python-packages/order_utils/setup.py

```diff
@@ -73,31 +75,64 @@ class CleanCommandExtension(clean):
     def run(self):
         """Run the regular clean, followed by our custom commands."""
         super().run()
-        rmtree("build", ignore_errors=True)
+        rmtree("dist", ignore_errors=True)
         rmtree(".mypy_cache", ignore_errors=True)
         rmtree(".tox", ignore_errors=True)
         rmtree(".pytest_cache", ignore_errors=True)
         rmtree("src/order_utils.egg-info", ignore_errors=True)
-        # delete all .pyc files
-        for root, _, files in walk("."):
-            for file in files:
-                (_, extension) = path.splitext(file)
-                if extension == ".pyc":
-                    remove(path.join(root, file))
+
+
+# pylint: disable=too-many-ancestors
+class TestPublishCommand(distutils.command.build_py.build_py):
+    """Custom command to publish to test.pypi.org."""
+
+    description = (
+        "Publish dist/* to test.pypi.org. Run sdist & bdist_wheel first."
+    )
+
+    def run(self):
+        """Run twine to upload to test.pypi.org."""
+        subprocess.check_call(  # nosec
+            (
+                "twine upload --repository-url https://test.pypi.org/legacy/"
+                + " --verbose dist/*"
+            ).split()
+        )
+
+
+# pylint: disable=too-many-ancestors
+class PublishCommand(distutils.command.build_py.build_py):
+    """Custom command to publish to pypi.org."""
+
+    description = "Publish dist/* to pypi.org. Run sdist & bdist_wheel first."
+
+    def run(self):
+        """Run twine to upload to pypi.org."""
+        subprocess.check_call("twine upload dist/*".split())  # nosec
+
+
+with open("README.md", "r") as file_handle:
+    README_MD = file_handle.read()
 
 
 setup(
-    name="order_utils",
-    version="1.0.0",
+    name="0x-order-utils",
+    version="0.1.0",
```

Comment:
Why did we have to change to version `0.1.0`? I'd rather we start at v1 due to SemVer rules about pre-1.0 releases.

Reply:
We didn't HAVE to, I just did it because it felt right. I was thinking that we would bump to 1.0 upon completion of what we set out as the MVP in our spec. I'm no expert in sever, but I just google it and it seems like my idea is consistent with the rules in its spec: "Major version zero (0.y.z) is for initial development. Anything may change at any time. The public API should not be considered stable."  Having said all that, I don't have strong feelings about it, so I'm happy to do whatever you think is right.

Reply:
Ah, I see. Ok, that's fine then.